content/applications/general/payment_acquirers/stripe.rst

```diff
@@ -56,12 +56,22 @@ endpoint** in your **Hosted endpoints**. A form opens, where you'll need to add
 When you click on **Add endpoint**, your Webhook is configured. You can then click on **reveal** to
 display your signing secret.
 
+.. _stripe/pmt:
+
 Enable local payment methods
 ----------------------------
 
 Local payment methods are payment methods that are only available for certain merchants and
 customers countries and currencies.
 
+Odoo 15 supports the following local payment methods:
```

Comment:
No need to indicate the version as this would quickly be out of date.
The user knows - hopefully - that they're browsing the v 15 of the documentation.

```suggestion
Odoo supports the following local payment methods:
```